src/helpers/format-price.ts

```diff
@@ -0,0 +1,22 @@
+import i18n from '@/setup/i18n';
+
+/**
+ * Formats the price according to project standards (Price value in cents).
+ */
+export default function formatPrice(value: string, currencyBefore: boolean, currencyAfter: boolean): string | null {
+  const locale = 'de-CH'; // TODO - replace hardcoded locale
+  const currency = 'CHF'; // TODO - replace hardcoded currency
+
+  const valueParsed = parseFloat(value);
```

Comment:
I would recommend we change the `value` type to number and remove the parsing. Then we can also drop the early exit.